sdk/storage/storage-blob/ChangeLog.md

```diff
@@ -3,6 +3,7 @@
 ## TBD 12.0.1
 
 - Updated to use OpenTelemetry 0.2 via `@azure/core-tracing`
+- Bug Fix - automatically convert empty prefixes (`""`) given as options to the `listContainers`, `listBlobsFlat`, and `listBlobsByHierarchy` methods into `undefined` to avoid an Error
```

Comment:
Sentense should end with `.`
```suggestion
- Bug Fix - automatically convert empty prefixes (`""`) given as options to the `listContainers`, `listBlobsFlat`, and `listBlobsByHierarchy` methods into `undefined` to avoid an Error.
```